### ISSUE TITLE
[BUG] Could Not Find App ID Qobuz

### ISSUE THREAD

**Issue by COTYWALT** [nathom/streamrip#333]
### Describe the bug

PS C:\Windows\system32> rip url https://play.qobuz.com/artist/4126621
Logging into qobuz

  Exception

  Could not find app id.

  at c:\users\cotyw\appdata\local\programs\python\python38\lib\site-packages\streamrip\spoofbuz.py:53 in get_app_id
       49│         match = re.search(self.app_id_regex, self.bundle)
       50│         if match is not None:
       51│             return str(match.group("app_id"))
       52│
    →  53│         raise Exception("Could not find app id.")
       54│
       55│     def get_secrets(self) -> List[str]:
       56│         """Get secrets."""
       57│         seed_matches = re.finditer(self.seed_timezone_regex, self.bundle)

If this was unexpected, please open a Bug Report at https://github.com/nathom/streamrip/issues/new/choose
PS C:\Windows\system32>

### Command Used

```bash
rip url https://play.qobuz.com/artist/4126621
```

### Debug Traceback

```text
see text
```

### Config File

```toml
default fresh config
```

### Operating System

Windows 10

### streamrip version

dev and stable

**Comment by xcvzn** [comment]
I have the same issue in a fresh install. I'm using Pop_Os

**Comment by snaplx** [comment]
> r'on/0.2/"},n.base_url="https://(nightly-){0,1)play.qobuz.com"'

error

  unbalanced parenthesis at position 163

  at ~\AppData\Local\Programs\Python\Python39\lib\sre_parse.py:969 in parse
       965│     p.state.flags = fix_flags(str, p.state.flags)
       966│
       967│     if source.next is not None:
       968│         assert source.next == ")"
    →  969│         raise source.error("unbalanced parenthesis")
       970│
       971│     for g in p.state.grouprefpos:
       972│         if g >= p.state.groups:
       973│             msg = "invalid group reference %d" % g

If this was unexpected, please open a Bug Report at https://github.com/nathom/streamrip/issues/new/choose

- Can you please help me with this error?

**Comment by nathom** [comment]
Fixed in `v1.9.5`
